cfgov/v1/jinja2/v1/includes/molecules/hero.html

```diff
@@ -62,7 +66,11 @@
                 {{- ' m-hero__jumbo' if value.is_jumbo else '' }}">
     <div class="m-hero_wrapper">
         <div class="m-hero_text">
-            <h1 class="m-hero_heading">{{ value.heading | safe }}</h1>
+            <h1 class="m-hero_heading">{{ value.heading | safe }}
+                {% if value.heading_with_line_break %}
+                <br /> {{ value.heading_with_line_break }}
```

Comment:
This works, but it could be shortened to `<br>` if preferred (since we're not in strict XHTML).

Reply:
changed in https://github.com/cfpb/consumerfinance.gov/pull/8191/commits/374073de5799829c008bc8e4281eddfd4d57220a